recipes/nuitka/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py install
```

Comment:
You can remove the `build.sh` file (but keep the `bld.bat`) and add a `build/script` in the metatadata. See below.